### PR TITLE
Some logicop quickrefs added in core for issue #179

### PR DESCRIPTION
**Pull request by nvandervoort**

### CODE REVIEW

**Comment by Gamrix** [comment]
The following ops are missing `wrxcsm@`. `==` should be `=`

File: pyrtl/core.py [hunk 1]
```diff
@@ -29,12 +29,32 @@ class LogicNet(collections.namedtuple('LogicNet', ['op', 'op_param', 'args', 'de
...
+    'or' and 'not', arithmetic operations such as '+' and '-', as well
     as other operations such as Memory ops, and concat, split, wire,
     and reg logic.
 
     The details of what is allowed in each of these fields is defined
-    in the comments of Block, and is checked by block.sanity_check
+    in the comments of Block, and is checked by `block.sanity_check`
+
+
+    `Operators`::
```
Comment (Gamrix) [inline pyrtl/core.py]: `Operators` should be renamed to `Logic Operations`

**Comment by nvandervoort** [comment]
^^ should be "...(still missing some)"

File: pyrtl/core.py [hunk 2]
```diff
@@ -29,12 +29,43 @@ class LogicNet(collections.namedtuple('LogicNet', ['op', 'op_param', 'args', 'de
...
+        ('=', None, (a1, a2), (out)) => check a1 & a2 equal, put result into out (0 | 1)
+        ('<', None, (a1, a2), (out)) => check a2 greater than a1, put result into out (0 | 1)
+        ('>', None, (a1, a2), (out)) => check a1 greater than a2, put result into out (0 | 1)
+        ('x', None, (x, a1, a2), (out)) => mux: connect a1 (x=0), a2 (x=1) to out;
+                                           x must be one bit; len(a1) = len(a2)
+        ('c', None, (*args), (out)) => concatenates *args (wires) into single WireVector;
+                                       puts first arg at MSB, last arg at LSB
+        ('s', (sel), (wire), (out)) => selects bits frm wire based on sel (std slicing syntax),
+                                       puts into out
+        ('r', None, (r1), None) => on pos clk edge copies inp of Register r1 to its outp
```
Comment (Gamrix) [inline pyrtl/core.py]: typo: `outp`

**Comment by Gamrix** [comment]
`w` is still missing

**Comment by Gamrix** [comment]
check `r` and `w` again.

**Comment by nvandervoort** [comment]
What's wrong with them? They're almost word-for-word the same as the documentation in `Block`

**Comment by Gamrix** [comment]
Hint:
```python
        if net.op in 'w~&|^nr' and net.dests[0].bitwidth > net.args[0].bitwidth:
            raise PyrtlInternalError('error, upper bits of destination unassigned')
```

**Pull request merged**